webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/angular/nodeProperties.js

```diff
@@ -94,10 +95,8 @@ app.controller('nodePropertiesCtrl', function ($scope, $http, DTOptionsBuilder,
         DTColumnDefBuilder.newColumnDef(2).withOption("sWidth",'5%')
     ];
   var currentNodeId
-  $scope.init = function(properties, nodeId, right){
-    if(!right){
-      $scope.columns[2].notVisible();
```

Comment:
You could have just changed the 2 to a 3 here :)

Reply:
the tab disapear when I used column 3

Reply:
The tab disapear whan I used column 3